modules/scalacheck/test/src/weaver/scalacheck/PropertyDogFoodTest.scala

```diff
@@ -115,23 +119,31 @@ object Meta {
     }
   }
 
-  object ConfigOverrideChecks extends ParallelChecks {
+  trait DiscardedChecks extends MetaSuite {
 
-    override def partiallyAppliedForall: PartiallyAppliedForall =
-      forall.withConfig(super.checkConfig.copy(
-        perPropertyParallelism = 1,
-        minimumSuccessful = 5))
+    test("Discards all the time") {
+      partiallyAppliedForall(Gen.posNum[Int].suchThat(_ < 0))(succeed)
+    }
   }
 
-  object DiscardedChecks extends SimpleIOSuite with Checkers {
+  object DiscardedChecks extends DiscardedChecks {
+
+    override def partiallyAppliedForall: PartiallyAppliedForall = forall
 
     override def checkConfig =
       super.checkConfig.copy(minimumSuccessful = 100,
                              // to avoid overcounting of discarded checks
                              perPropertyParallelism = 1)
+  }
 
-    test("Discards all the time") {
-      forall(Gen.posNum[Int].suchThat(_ < 0))(succeed)
-    }
+  object ConfigOverrideChecks extends DiscardedChecks {
```

Comment:
ditto on the arbitrary reorder

Reply:
Fixed in https://github.com/disneystreaming/weaver-test/pull/631/commits/56d22e637300f12339a628a4646346ed28877813